packages/webdriverio/tests/commands/browser/url.test.js

```diff
@@ -27,14 +27,39 @@ describe('url', () => {
             .toEqual({ url: 'http://foobar.com/foobar' })
     })
 
-    it('should throw an exception when a non-string value passed in', async () => {
+    it('should throw an exception when a non-string value passed in path', async () => {
         try {
             await browser.url(true)
         } catch (e) {
             expect(e.message).toContain('command needs to be type of string')
         }
     })
 
+    it('should throw an exception when a non-function value passed in inject', async () => {
```

Comment:
This test doesn't fail if no error was thrown, instead use `expect(X).reject.toThrowError(Y)`

Reply:
As we are passing a wrong type, it will throw an error and that's what we are doing here.

```
it('should throw an exception when a non-function value passed in inject', async () => {
try {
            await browser.url(``, {inject: 123}) // inject only accepts function and rest will return error and we are validating it in the catch block
        } catch (e) {
            expect(e.message).toContain('command needs to be type of string')
        }
    })
```

Reply:
What I mean is that it is a false positive if (for whatever reason) no error is thrown.

Reply:
The solution here is to add

```js
expect.assertions(1)
```

to say we at least expect 1 assertion. This hasn't to do anything with your patch but we should fix it while we at it.